pyproject.toml

```diff
@@ -11,3 +11,18 @@ build-backend = "setuptools.build_meta"
 required-version = 23
 target-version = ["py37", "py38", "py39", "py310", "py311", "py312"]
 safe = true
+
+[tool.pytest.ini_options]
+minversion = "6.0"
+python_files = [
+    "test/test*.py",
+    "test/Test*.py",
+]
+python_classes = "Test"
+python_functions = "test"
+filterwarnings = [
+    "error",
+    # note the use of single quote below to denote "raw" strings in TOML
+    'ignore:tostring\(\) is deprecated. Use tobytes\(\) instead\.:DeprecationWarning:OpenGL',
+    'ignore:Jupyter is migrating its paths to use standard platformdirs:DeprecationWarning',
```

Comment:
Listing here general warnings from our dependencies

Reply:
For the jupyter path warnings you do not specify the module but I guess there are several? I don't remember but in our CI I do this instead of ignoring the warnings:

```
test-3.11:
  extends: .test-3.11_glx
  variables:
    JUPYTER_PLATFORM_DIRS: 1
```

I'm fine with it either way.

Reply:
AFAIR, it occurs once through an import, during the test collections I think.
But indeed, the env var does the job

Reply:
I'll keep it like that for now: it's set at a single place instead of appveyor+github configs, and also works when running locally.